publication-commons/src/test/java/no/unit/nva/publication/service/impl/PublishingServiceTest.java

```diff
@@ -279,7 +280,7 @@ void shouldPersistFilesApprovalThesisForChannelClaimInstitutionWhenPublicationTo
 
     @Test
     void shouldPersistDoiRequestWhenPublishingPublicationWithDoi() throws ApiGatewayException {
-        var resource = persistResource(Resource.fromPublication(randomPublication().copy().withStatus(DRAFT).build()));
+        var resource = persistResource(Resource.fromPublication(randomNonDegreePublication().copy().withStatus(DRAFT).build()));
```

Comment:
Some of these tests are also failing sporadically because of publication type degree. Degrees are not relevant in these test cases, testing with non degrees.